pkg/webhook/internal/cert/writer/secret_test.go

```diff
@@ -78,6 +79,10 @@ var _ = Describe("secretCertWriter", func() {
 			//isController := true
 			//blockOwnerDeletion := true
 			secret = &corev1.Secret{
+				TypeMeta: metav1.TypeMeta{
+					APIVersion: "v1",
+					Kind:       "Secret",
+				},
```

Comment:
As per my understanding, as long as the client has correct scheme (in this case scheme for core-types which is set by default), we don't need to set typemeta. Has anything changed ?